### DISCUSSION TITLE
GitHub Support does not respond even after 15 days

### DISCUSSION THREAD

**Discussion by bozkayasalihx** [community/community#60944] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Yes: GitHub Support does not respond even after 15days.

I can’t get support from community about my issue; It’s about account being flagged without no reason. I think its more like GitHub bot issues that sees me as a bot, clearly i'm not
so please help me out ASAP

How long did it take to respond when you created a ticket on GitHub? Is this general timeframe?

**Comment by LiteBrite82** [comment, marked as answer]
Hi @bozkayasalihx 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
